build-system/tasks/check-types.js

```diff
@@ -52,6 +57,76 @@ const getExtensionSrcPaths = () =>
  * @type {Object<string, Object|function():Object>}
  */
 const TYPE_CHECK_TARGETS = {
+  // Below are targets containing individual directories which are fully passing
+  // type-checking. Do not remove or disable anything on this list.
+  // Goal: Remove 'QUIET' from all of them.
+  // To test a target locally:
+  //   `amp check-types --target=src-foo-bar --warning_level=verbose`
+  'src-amp-story-player': {
+    srcGlobs: ['src/amp-story-player{,/**}/*.js'],
+    warningLevel: 'QUIET',
+  },
+  'src-context': {
+    srcGlobs: ['src/context{,/**}/*.js'],
+    warningLevel: 'QUIET',
+  },
+  'src-core': {
+    srcGlobs: ['src/core{,/**}/*.js'],
+    externGlobs: [CORE_EXTERNS_GLOB],
+  },
+  'src-examiner': {
+    srcGlobs: ['src/examiner{,/**}/*.js'],
+    warningLevel: 'QUIET',
+  },
+  'src-experiments': {
+    srcGlobs: ['src/experiments{,/**}/*.js'],
+    warningLevel: 'QUIET',
+  },
+  'src-inabox': {
+    srcGlobs: ['src/inabox{,/**}/*.js'],
+    warningLevel: 'QUIET',
+  },
+  'src-polyfills': {
+    srcGlobs: ['src/polyfills{,/**}/*.js'],
+    warningLevel: 'QUIET',
+  },
+  'src-polyfillstub': {
```

Comment:
nit: I'd combine this with src-polyfills

Reply:
Two steps ahead of ya: https://github.com/ampproject/amphtml/pull/34020
Will update whichever one gets submitted second.